File: submissions/cse2015078/problem2.cpp
#include<iostream>
#include<vector>
#include<algorithm>

using namespace std;

int main()
{
    int N,C,I;
    cin >> N;
    int cnt = 1;
    for(int i=0; i<N;i++){
        cin >> C;
        cin >> I;
        vector <int> list;
        for(int j=0;j<I;j++){
            int a;
            cin >>a;
            list.push_back(a);
        }
        int flag = 0,l = 0,k=1;

        int se = list.size();
        while((l < se)&&flag == 0){
            while((l+k < se)&&flag == 0){
                    //cout <<"I ran " <<(l)<<endl;
                if(list[l]+list[l+k] == C){
                    //cout << "Case #"<<cnt<<":";
                    cnt++;
                    cout << (l+1) << " "<< (l+k+1) <<endl;
                    flag = 1;
                } else {
                    k++;
                }
            }
            l++;
            k = 1;
        }
        if(cnt == 1){
            cout <<"NA"<<endl;
        }
    }
    return 0;
}

File: submissions/aniket98/problem2.java
#include <iostream>
using namespace std;
int main()
{
    int N[100],T,i,x=0,y=0;
    do{cin>>T;}
    while(T<1&&T>100);
    for(i=0;i<T;i++)
    {
     do{cin>>N[i];}
     while(N[i]<1&&N[i]>1000000000);
    }
    for(i=0;i<T;i++)
    {
     if(N[i]%4==1)
     {x=x+N[i];y=y+0;}
     if(N[i]%4==2)
     {x=x+0;y=y+N[i];}
     if(N[i]%4==3)
     {x=x-N[i];y=y+0;}
     if(N[i]%4==0)
     {x=x+0;y=y-N[i];}
     cout<<x<<y;
    }
    return 0;
}
